proofs/cbmc/keccakf1600_finalize/Makefile

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
These proof directories all appear to be mis-named. There is no function called "keccakf1600_init" for example, but there is "keccak_init". Same comment for other directories here. Simiarly, I think the directory "keccakf1600_permute" should be "keccakf1600_statepermute"?

Reply:
Thank @manastasova, but I don't see the fixed code yet?